Client/src/Components/DataCell/DataCell.js

```diff
@@ -87,8 +94,24 @@ export default function DataCell() {
                             <input type="file" id="jsonFile" accept="application/json" />
                             <Button type="submit" variant="contained"> Upload this file! </Button>
                         </div>
+
                     </form>
                 </Box>
+
+                <Box  p={4}>
+                    <div>
+                        {/**for downloading sample empty json file*/}
+                         <Download file= "emptyJason.json" content= {JSON.stringify(emptyJSFile,null,2)}>
+                            <Button type="submit" variant="contained" onClick={console.log('success fully json downloaded')}> Download Sample JSON file </Button>
+                         </Download>                                                               
+                    </div>
+                    {/**for downnloading instructions readMe for users */}
+                    <div>
+                        <Download file="readMe.txt" content = {readme + fields + info}>
+                             <a href= "http://localhost:3000/#/uploadFile"> ReadME </a>  

```

Comment:
What is this API call for?

Reply:
we also gonna have to deal with the hardcoded URL in the future, maybe put it in env

Reply:
if i don't put the href, the page changes by default to our front page. this prevent the page from changing